### ISSUE TITLE
Content Returning Blank

### ISSUE THREAD

**Issue by asianshyguy** [IonicaBizau/bible-scraper#3]
After attempting the documented script on the tutorial (below)
```
const kjv = new BibleScraper(BibleScraper.TRANSLATIONS.KJV);
kjv.verse("1CO.13.4").then(console.log);
```

... I was returned an empty object
`{ content: '', reference: '' }`

**Comment by corneal64** [comment]
Same issue here! 

`{ content: '', reference: '' }
`

**Comment by grnsmn** [comment]
Same issue!

Someone know how to solve?

**Comment by IonicaBizau** [comment]
Should be fixed now, in 2.0.0. Thanks @grnsmn!
